lib/activity_reporter.rb

```diff
@@ -0,0 +1,43 @@
+# generates text for a report on druid count and logs
+class ActivityReporter
+  DRUID_REGEXP = '[[:lower:]]{2}\\d{3}[[:lower:]]{2}\\d{4}'.freeze
```

Comment:
while this works, it is strictly speaking incorrect.  I expect you already know that.  See https://github.com/sul-dlss/druid-tools/blob/master/lib/druid_tools/druid.rb#L19-#L20. 

(no need to change it - I think this is fine here -- apparently I'm just being annoying.)